org.flightgear.FlightGear.yaml

```diff
@@ -108,13 +121,13 @@ modules:
     sha256: 277e817a39edb917976441a808b750ee64be38e077897d0d77c55dfc8eba9640
     dest: fgdata
   build-commands:
-  - mkdir --parents /app/share/flightgear
-  - mv fgdata/* /app/share/flightgear/
-    
+  - mkdir --parents ~/.fgfs
+  - mv fgdata/* ~/.fgfs/
```

Comment:
```suggestion
  - cp fgdata/* ~/.fgfs/
```

Reply:
Hi, I don't know why it fails with: `mv: target '/srv/buildbot/.fgfs/' is not a directory`.
Given that the error is on the target of the `mv` command, I don't think `cp -r` may solve the problem. Strange that the folder `~/.fgfs` is not created...

Reply:
We don't have much to lose trying it after all…